lib/components.js

```diff
@@ -363,7 +363,9 @@ class DatePicker extends Component {
       'minimumDate',
       'minuteInterval',
       'mode',
-      'timeZoneOffsetInMinutes'
+      'timeZoneOffsetInMinutes',
+      'background',
```

Comment:
I mean, the typical usage of a `config` option from my point of view would be to put all the configuration fields specific to the template at hand. `background` being a prop of `TouchableNativeFeedback` seems specific to `datepicker.android.js`